extensions/2.0/Khronos/KHR_materials_clearcoat/README.md

```diff
@@ -0,0 +1,92 @@
+# KHR\_materials\_clearcoat
+
+## Khronos 3D Formats Working Group
+
+* Norbert Nopper, UX3D [@UX3DGpuSoftware](https://twitter.com/UX3DGpuSoftware)
+* Tobias Haeussler, Dassault Systemes [@proog128](https://github.com/proog128)
+* Alexey Knyazev [@lexaknyazev](https://github.com/lexaknyazev)
+* Don McCurdy, Google [@donrmccurdy](https://twitter.com/donrmccurdy)
+* Sebastien Vandenberghe, Microsoft [@sebavanjs](https://twitter.com/sebavanjs)
+* Romain Guy, Google [@romainguy](https://twitter.com/romainguy)
+* Ed Mackey, AGI [@emackey](https://twitter.com/emackey)
+* Alex Wood, AGI [@abwood](https://twitter.com/abwood)
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension defines a clear coating that can be layered on top of an existing glTF material definition.  A clear coat is a common technique used in Physically-Based Rendering to represent a protective layer applied to a base material.  See [Theory, Documentation and Implementations](#theory-documentation-and-implementations)
+
+## Extending Materials
+
+The PBR clearcoat materials are defined by adding the `KHR_materials_clearcoat` extension to any glTF material. 
```

Comment:
The phrase "`to any glTF material`" is probably more open-ended than it should be. Is it reasonable to specify interaction with other material extensions (`KHR_materials_unlit`, `KHR_materials_pbrSpecularGlossiness`) here, or what would be the right place? 

My preference would be that neither the unlit nor spec/gloss extensions be used with clearcoat. The first is fairly obvious. The second is more opinionated, but (in my opinion) the overall goals of [advanced PBR](https://github.com/KhronosGroup/glTF/milestone/2) in glTF would be better served by focusing on a single PBR workflow. Clearcoat — and future advanced PBR extensions — should gradually obviate `KHR_materials_pbrSpecularGlossiness`. 

Whether that means normatively disallowing use of the two extensions together, or just informal guidance, I'm not sure.

Reply:
I made some updates, how does it look?

Reply:
The change matches my preference, thanks. If others are concerned about this choice, please chime in!